docs/site/tutorials/device_placement.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>The use of a `get` prefix in function names is generally discouraged in Swift. The Swift API Design Guidelines state:</p><blockquote>Name functions and methods according to their side-effect.</blockquote><p><br></p><p>`get` should be removed from these names.</p><p><br></p><p>For more details on naming, see <a href="https://swift.org/documentation/api-design-guidelines/#promote-clear-usage" target="_blank">Swift API Design Guidelines - Promote Clear Usage</a>.</p><p><br></p><p><br></p><p>Moreover, the `getTimeString` function can be renamed to be more clear and not to repeat the return type information. And the body of the `getTimeString` function can be improved to use early return, or even better, pattern matching.</p><p><br></p><p>```swift</p><p>func timeDescription(_ nanoseconds: Double) -&gt; String {</p><p>switch nanoseconds {</p><p>case ..&lt;1e3:</p><p>return "\(nanoseconds) ns"</p><p>case ..&lt;1e6:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;return "\(nanoseconds/1e3) µs"</p><p>case ..&lt;1e9:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;return "\(nanoseconds/1e6) ms"</p><p>default:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;return "\(nanoseconds/1e9) seconds"</p><p>}</p><p>}</p><p>```</p><p><br></p><p><br></p><p>In your first function, please avoid the use of&nbsp;explicit `Array&lt;T&gt;`. Also, prefer type inference and literal conversion over explicit calls to the initializer.</p><p>```swift</p><p>var timings: [Double] = 0.0</p><p>```</p><p>For more details on types with short hand names like array and dictionary, see the <a href="https://google.github.io/swift/#types-with-shorthand-names" target="_blank">Google Swift Style Guide</a>.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Ah, of course! This should be corrected now.

Reply:
Got it.